add_to_app/AndroidAddToAppDemo/app/build.gradle

```diff
@@ -24,6 +24,12 @@ android {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
             signingConfig signingConfigs.debug
+            packagingOptions {
+                doNotStrip "*/arm64-v8a/*.so"
+                doNotStrip "*/armeabi-v7a/*.so"
+                doNotStrip "*/x86/*.so"
+                doNotStrip "*/x86_64/*.so"
+            }
```

Comment:
I don't think you want this. I think you only want to not strip libapp.so.

Does `packagingOptions { "libapp.so" }` work?

Reply:
https://stackoverflow.com/questions/52972371/set-donotstrip-packagingoptions-to-a-specific-buildtype

Reply:
Fixed

Reply:
Looks like maybe:

https://developer.android.com/reference/tools/gradle-api/8.2/com/android/build/api/dsl/JniLibsPackaging#keepDebugSymbols()

`android.packagingOptions.jniLibs.keepDebugSymbols += "**/libapp.so"`